ui/app/AppLayouts/Chat/panels/GroupChatPanel.qml

```diff
@@ -0,0 +1,180 @@
+import QtQuick 2.13
+import QtQuick.Layouts 1.13
+
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+
+import utils 1.0
+
+
+RowLayout {
+     id: root
+
+     property var sectionModule
+     property var chatContentModule
+     property var rootStore
+     property int maxHeight
+
+     signal panelClosed()
+
+     QtObject {
+         id: d
+
+         property ListModel groupUsersModel: ListModel { }
+         property ListModel contactsModel: ListModel { }
+         property var addedMembersIds: []
+         property var removedMembersIds: []
+
+         function initialize () {
+             groupUsersModel.clear()
+             contactsModel.clear()
+             addedMembersIds = []
+             removedMembersIds = []
+             tagSelector.namesModel.clear()
+         }
+     }
+
+     ListView {
+         id: groupUsersModelListView
+         visible: false
+         model: root.chatContentModule.usersModule.model
+         delegate: Item {
+             property string publicId: model.id
+             property string name: model.name
+             property string icon: model.icon
+             property bool isAdmin: model.isAdmin
+         }
+     }
+
+     ListView {
+         id: contactsModelListView
+         visible: false
+         model: root.rootStore.contactsModel
+         delegate: Item {
+             property string publicId: model.pubKey
+             property string name: model.name
+             property string icon: model.icon
+         }
+     }
+
+     clip: true
+
+     Component.onCompleted: {
+         d.initialize()
+
+         // Build groupUsersModel type from model type (to fit with expected StatusTagSelector format
```

Comment:
What happens when the source model is changed? For e.g. contact is added or removed? Will it auto rebuild `groupUsersModel` ? :thinking:

Reply:
> What happens when the source model is changed? For e.g. contact is added or removed? Will it auto rebuild `groupUsersModel` ? 🤔

An issue was created here to keep track of that: https://github.com/status-im/status-desktop/issues/5956